corporate/lib/stripe.py

```diff
@@ -826,14 +826,16 @@ def get_stripe_customer_portal_url(
         if return_to_billing_page:
             return_url = f"{self.billing_session_url}/billing/"
         else:
-            assert tier is not None
             base_return_url = f"{self.billing_session_url}/upgrade/"
-            params = {
-                "manual_license_management": str(manual_license_management).lower(),
-                "tier": str(tier),
-                "setup_payment_by_invoice": str(setup_payment_by_invoice).lower(),
-            }
-            return_url = f"{base_return_url}?{urlencode(params)}"
+            if tier is None:  # nocoverage
+                return_url = base_return_url
```

Comment:
The upgrade page logic can get a bit off if these parameters are missing. Would it make sense for the redirect to be to the billing page or the plans page if the tier is missing? Thinking about the case when the person who does type in the URL manually is a org owner or billing admin.

Reply:
Okay, changed it to redirect it to billing page if tier is None.